### ISSUE TITLE
NaN or Inf found in input tensor

### ISSUE THREAD

**Issue by JianyyuWang** [open-mmlab/OpenPCDet#1006]
I find that if anchor_head_multi.py was used on nuscenes and RCNN is used, it will cause an error like NaN or Inf found in input tensor.
Does Someone know what casues that ?

**Comment by OrangeSodahub** [comment]
Also occurred this, have you solved it ?

**Comment by JianyyuWang** [comment]
> 

not yet, I tried to change spcnv2.X to spconv 1.X ,but failed

**Comment by OrangeSodahub** [comment]
I've solved it. Make sure that label files are not empty!

**Comment by JianyyuWang** [comment]
> 

I do not quite understand . the label files is the official, you mean that we need to clear away the empty label files?

**Comment by JianyyuWang** [comment]
> I've solved it. Make sure that label files are not empty!

And I checked the label files. when error occured, the label files are not empty

**Comment by OrangeSodahub** [comment]
@JianyyuWang  I used my own custom dataset. For my dataset, it exactly helped.

**Comment by TaigaMoriguchi** [comment]
I was able to solve this by updating the learning rate.

**Comment by OuYaozhong** [comment]
Same problem. Anybody solved it ? Just use the official nuScene and follow the steps of INSTALL.md.

**Comment by SimonSongg** [comment]
Same issue, I've clean up the samples without object. 

> Same problem. Anybody solved it ? Just use the official nuScene and follow the steps of INSTALL.md. @JianyyuWang

Although I used my own dataset, I met this issue when using CenterPoint. Finally I found there were some annotation issue: the height of the bounding box of some targets were negative. And the 3-dim of the bounding box takes .log(). Python returns NaN when trying to take logarithm of a negative number.

I hope this helps.
